react-frontend/src/profile/profile.css

```diff
@@ -40,5 +40,44 @@
 }
 
 .profile-picture:hover {
-  filter: brightness(0.5); /* Darken on hover */
+  filter: brightness(0.5); 
 }
+
+.collection-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.collection-link {
+  flex: 1; 
+}
+
+.bookmark-container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding-left: 10px;
+  padding-right: 20px;
+}
+
+.edit-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+.edit-neutral {
+  width: 24px;
+  height: 24px; 
+  transition: transform 0.2s ease;
```

Comment:
You're right, I could probably move display: flex, align-items: center to a new style set to remove duplicate code but then it would require me to add an extra reference to the new styling sheet in the html.